scripts/get_benchmark_results.py

```diff
@@ -6,12 +6,19 @@
 if __name__ == '__main__':
     output_bucket = os.getenv("OUTPUT_BUCKET")
 
+    days = os.getenv("NUMBER_OF_DAYS")
+    if days and days.isdigit() is False:
+        print("'NUMBER_OF_DAYS' environment variable must be a valid integer value")
+        sys.exit(1)
+
+    days = int(days) if days else None
```

Comment:
This should just reuse the existing method we have for that handles this env var.
```py
parsed_variables = parse_environment_variables()
number_of_days = parsed_variables['number_of_days']
```

Reply:
I've added this, although now it makes the `OUTPUT_DIR` env mandatory for this script